pysat/_instrument.py

```diff
@@ -68,6 +73,29 @@ class Instrument(object):
     name_label : str
         label to use for long name. Defaults to 'long_name' but some implementations
         will use 'Long_Name'
+    units_label : str
```

Comment:
I like the new meta values.  The default meta print could stand to be cleaner, but all of this information is useful.

Reply:
Thanks. 

I agree with you on the print statement. It is reduced from an earlier version (which just dumped pandas strings), gave way too much. This version not quite enough.